tools/quic/quic_transport_server.py

```diff
@@ -167,73 +167,29 @@ def pop(self, label: bytes) -> Optional[SessionTicket]:
         return self.tickets.pop(label, None)
 
 
-if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description='QUIC server')
-    parser.add_argument(
-        '-c',
-        '--certificate',
-        type=str,
-        required=True,
-        help='load the TLS certificate from the specified file',
-    )
-    parser.add_argument(
-        '--host',
-        type=str,
-        default='::',
-        help='listen on the specified address (defaults to ::)',
-    )
-    parser.add_argument(
-        '--port',
-        type=int,
-        default=4433,
-        help='listen on the specified port (defaults to 4433)',
-    )
-    parser.add_argument(
-        '-k',
-        '--private-key',
-        type=str,
-        required=True,
-        help='load the TLS private key from the specified file',
-    )
-    parser.add_argument(
-        '--handlers-path',
-        type=str,
-        required=True,
-        help='the directory path of QuicTransport event handlers',
-    )
-    parser.add_argument(
-        '-v',
-        '--verbose',
-        action='store_true',
-        help='increase logging verbosity'
-    )
-    args = parser.parse_args()
-
-    logging.basicConfig(
-        format='%(asctime)s %(levelname)s %(name)s %(message)s',
-        level=logging.DEBUG if args.verbose else logging.INFO,
-    )
-
+def start(kwargs):
     configuration = QuicConfiguration(
         alpn_protocols=['wq-vvv-01'] + ['siduck'],
         is_client=False,
         max_datagram_frame_size=65536,
     )
 
-    handlers_path = os.path.abspath(os.path.expanduser(args.handlers_path))
-    logging.log(logging.INFO, 'port = %s' % args.port)
-    logging.log(logging.INFO, 'handlers path = %s' % handlers_path)
+    global handlers_path
```

Comment:
Missing this was a bug that caused "Client indication error: unsupported operand type(s) for +: 'NoneType' and 'str'".

Reply:
Yeah, previously this code was in the global scope but now it's in a function.